app/components/UserGrid/index.tsx

```diff
@@ -26,17 +28,23 @@ const UserGrid = ({
       }px, 1fr))`,
       gridTemplateRows: users.length === 0 ? 0 : size + padding,
       overflow: 'hidden',
-      minHeight: minRows * size + (minRows - 1) * padding,
-      maxHeight: maxRows * size + (maxRows - 1) * padding,
+      minHeight: minRows * size + (minRows - 1) * padding + 3.3,
+      maxHeight: maxRows * size + (maxRows - 1) * padding + 3.3,
     }}
   >
     {users.map((user) => (
-      <RegisteredCell key={user.id} user={user} />
+      <RegisteredUserCell key={user.id} user={user} />
     ))}
+
+    {/* Only 15 users are passed into the users prop */}
+    {minRows > 0 &&
+      Array.from({ length: 15 - users.length }, (_, index) => (
```

Comment:
Why is this not something like `maxRows*5 (images pr row) - users-length`? Wouldn't 15 - users.length produce three rows of 5 images rather than two, or am I missing something here

Reply:
The amount of images per row is not always 5. Try to resize your viewport (to just before it converges to mobile view) and you'll see.